examples/monitoring-alert/README.md

```diff
@@ -0,0 +1,30 @@
+## Requirements
+
+No requirements.
```

Comment:
let's help customers consider how to best handle tf.state file.
1) consider sharing the best practices for tf.state file
2) Default should be to use a bucket via remote backend https://developer.hashicorp.com/terraform/language/backend

Reply:
You mean I should paste this URL into the requirements section?

Reply:
I think there is a better page on hashi docs that indicate best practices for the state file.  Plz find that.   Then, both can be provided.

Reply:
I found this page on hashi docs: https://developer.hashicorp.com/terraform/language/state. I couldn't find one specific for best practices on hashi docs. Could I use https://developer.hashicorp.com/terraform/language/backend or the one I sent here?